components/Sections/SectionFeed.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { compose } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import { Center, Interaction } from '@project-r/styleguide'
+
+import withT from '../../lib/withT'
+
+import Box from '../Frame/Box'
+import { onDocumentFragment as bookmarkOnDocumentFragment } from '../Bookmarks/fragments'
+import { WithoutMembership, WithActiveMembership } from '../Auth/withMembership'
+
+import DocumentListContainer from '../Feed/DocumentListContainer'
+
+const SectionFeed = ({ t, formats }) => {
+  const getFeedDocuments = `
+  query getBookmarkedDocuments($cursor: String) {
```

Comment:
```suggestion
  query getSectionDocuments($cursor: String, $formats: [String!]) {
```

Proper name & pass the formats as a proper variable. Better for backend query analysis.

Reply:
Good catch, thanks!